test/unit/rules/block-indentation-test.js

```diff
@@ -1327,5 +1429,255 @@ generateRuleTests({
         `);
       },
     },
+    {
+      template: [
+        "import { hbs } from 'ember-cli-htmlbars';",
+        '',
+        "test('it renders', async (assert) => {",
+        '  await render(hbs`',
+        '  <div class="parent">',
+        '    <div class="child"></div>',
+        '  </div>',
+        '  `);',
+        ');',
+      ].join('\n'),
+      fixedTemplate: [
+        "import { hbs } from 'ember-cli-htmlbars';",
+        '',
+        "test('it renders', async (assert) => {",
+        '  await render(hbs`',
+        '    <div class="parent">',
+        '      <div class="child"></div>',
+        '    </div>',
+        '  `);',
+        ');',
+      ].join('\n'),
+      meta: {
+        filePath: 'layout.js',
+      },
+
+      verifyResults(results) {
+        expect(results).toMatchInlineSnapshot(`
```

Comment:
Creating these snapshots was not trivial to do, is there a command for this?

Reply:
`jest --updateSnapshot` ?